infrastructure/aws/production/ecs-task-definition-task-processor.json

```diff
@@ -126,6 +126,10 @@
                 {
                     "name": "PIPEDRIVE_LEAD_LABEL_EXISTING_CUSTOMER_ID",
                     "value": "d828a3a0-bf34-11ed-b08d-b12d3497f5c4"
+                },
+                {
+                    "name": "TASK_RETENTION_DAYS",
+                    "value": "280"
```

Comment:
We originally released the task processor on 22/9/2023 meaning that we should have around 295 days of tasks in the database. Setting the retention period will allow us stop the amount of tasks increasing at least while slowly allowing us to chip away at clearing up old tasks. The plan will be to slowly reduce this retention period down to the 30 day default.